### DISCUSSION TITLE
When I try to search/filter on a string, it isn't searching my Alias Descriptions?

### DISCUSSION THREAD

**Discussion by TriggerDingus** [simple-login/app#2219] [Q&A] [answered]
I'm guessing I must be doing something incorrectly. I have an alias that I use for Namecheap and so I have that term in my alias description so I can find the alias. However, when I type "namecheap" in the Filter bar, it doesn't find it. This alias is a few years old, so this isn't a new alias that I just created. 

I've attached a pic of the alias description as well as a pic of the filter which won't find it.
![2024-09-15_11-44](https://github.com/user-attachments/assets/cb34b9db-1c59-4d30-9568-d2df91f78d02)
![2024-09-15_11-43](https://github.com/user-attachments/assets/426d7638-4339-4bd4-90a6-24643d01b194)

**Comment by nguyenkims** [comment, marked as answer]
There's a typo in the alias description, it's nameheap instead of namecheap
